src/components/save-modal/SaveModal.jsx

```diff
@@ -62,7 +78,81 @@ export const SaveModal = () => {
 
             <WatchLaterIcon className="m-0p5" />
           </li>
+
+          {playlistsStatus === "success" &&
+            playlistsData.map(({ _id, title }) => (
+              <li key={_id} className="align-i-ctr flex justify-c-sb mb-1">
+                <label htmlFor={_id} className="align-i-ctr cursor-ptr flex">
+                  <input
+                    type="checkbox"
+                    name="playlists"
+                    value={title}
+                    id={_id}
+                    className="cursor-ptr mr-1"
+                    checked={selectedPlaylists.includes(title)}
+                    onChange={(event) => {
+                      handlePlaylistsCheckboxDispatch(event, _id);
+                    }}
+                  />
+                  {title}
+                </label>
+
+                <PlaySpeedIcon className="m-0p5" />
+              </li>
+            ))}
         </ul>
+
+        <div
+          className={classNames("modal-foot", "text-left", styles.modalFoot)}
+        >
+          {showNewPlaylistForm === false ? (
```

Comment:
for boolean values, you can use "!" sign for handling in app. For example, here ,you can use " !showPlaylistForm" instead of comparison.

Reply:
Thanks, will look into this.